backend/entrypoint.prod.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
```

Comment:
Czemu mamy inny entrypoint dla produkcji? Chyba w obu staramy się to samo osiągnąć, nie?